datalad/cli/common_args.py

```diff
@@ -34,10 +34,13 @@
         ('-c',),
         dict(action='append',
              dest='cfg_overrides',
-             metavar='KEY=VALUE',
-             help="""configuration variable setting. Overrides any
-             configuration read from a file, but is potentially overridden
-             itself by configuration variables in the process environment.
+             metavar='[:]name[=value]',
+             help="""specify configuration setting overrides. They override any
+             configuration read from a file. A configuration can also be
+             unset temporarily by prefixing its name with a colon (':') and
+             passing that as a value, e.g. ':user.name'.
```

Comment:
I am not sure what that "passing that as a value" means, could it just be

```suggestion
             unset temporarily by prefixing its name with a colon (':'), e.g. ':user.name'.
```

?

Reply:
As a value to the option this help is about (`-c`). But I agree it may be confusing as the value itself is described as `name=value`.